playground/frontend/lib/pages/embedded_playground/components/embedded_actions.dart

```diff
@@ -58,16 +57,12 @@ class EmbeddedActions extends StatelessWidget {
   void _openStandalonePlayground(PlaygroundController controller) {
     // The empty list forces the parsing of EmptyExampleLoadingDescriptor
     // and prevents the glimpse of the default catalog example.
-    final window = html.window.open(
-      '/?$kExamplesParam=[]&$kSdkParam=${controller.sdk?.id}',
-      '',
-    );
 
-    javaScriptPostMessageRepeated(
-      window,
-      SetContentMessage(
-        descriptor: controller.getLoadingDescriptor(),
-      ),
+    var descriptors = controller.getLoadingDescriptor().toJson()['descriptors'];
+    var json = jsonEncode(descriptors);
+    html.window.open(
+      '/?$kExamplesParam=$json&$kSdkParam=${controller.sdk?.id}',
+      '',
```

Comment:
This would exceed the URL length of browser standards.

Reply:
fixed